.github/workflows/qc.yml

```diff
@@ -51,3 +51,11 @@ jobs:
 
       - name: Test with pytest and generate coverage file
         run: poetry run tox -e py
+
+      #----------------------------------------------
+      # QC with pandas 1.x.x
+      #----------------------------------------------
```

Comment:
add a comment that says that we run `poetry update` frequently so the lock file should in theory have an up to date >2 vesion, which gives us confidence we are testing against 1 and 2